packages/react-tracking-pixel/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "@shopify/react-tracking-pixel",
+  "version": "0.1.0",
```

Comment:
Should this be 1.0.0?  Seems ready for primetime.

Reply:
Yeah should be ready for release 1.0.0